src/button.rs

```diff
@@ -0,0 +1,53 @@
+use rppal::gpio::{Error, Gpio, InputPin, Level};
+use std::cmp::Ordering::{Equal, Greater, Less};
+use std::time::Duration;
+use tokio::spawn;
+use tokio::sync::broadcast;
+use tokio::time::sleep;
+use tracing::{debug, error, info};
+
+pub struct Button {
+    pub receiver: broadcast::Receiver<()>,
+}
+
+impl Button {
+    const BUTTON_POLLING_MAX_COUNT: u64 = 2;
+    const BUTTON_POLLING_INTERVAL_MILLIS: u64 = 1000;
```

Comment:
Would go down to some few hundred millis, otherwise, one might think the button doesn't do anything at all.